### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by peterboro**
In this pull request I implemented the following changes:
- Added a contact form to index.html file.
- Added styling to the contact form in the style.css file.
- Ran linters.
- Changes also made by collaborator

### CODE REVIEW

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @peterboro , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
...
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -263,5 +263,25 @@ <h1 class="first-title2">About Myself</h1>
         </div>
     </div>
 </section>
+<footer class="footer" id="contact">
+    <div class="container-3">
+        <div class="parent">
+            <img src="./Images/contactformshapes-mobile.png" class="over-img" alt="contactformimage"/>
```
Comment (gtekle) [inline index.html:269]: - [ Optional] This is a decorator image. Therefore, you should add it as a css background image to the footer instead.

File: index.html [hunk 2]
```diff
@@ -263,5 +263,25 @@ <h1 class="first-title2">About Myself</h1>
...
+        </div>  
+    <h1 class="first-title3">Contact me</h1>
```
Comment (gtekle) [inline index.html:271]: - [ Optional] You should use only one `<h1>` heading tag in an HTML document for semantic reasons. And that one should go to the most important heading the HTML page. This helps for SEO and accessibility for the page.

File: index.html [hunk 3]
```diff
@@ -263,5 +263,25 @@ <h1 class="first-title2">About Myself</h1>
...
+    <p class="description6">
+		If you have an application you are interested in developing,a feature that you need built or a project that needs coding. I’d love to help with it
+    </p>
+    <form class="contactform" action="https://formspree.io/f/myyoqzdn" method="POST">
+		<input type="text" name="name" placeholder="Full Name" maxlength="30" required />
+		<input type="email" name="email" placeholder="Email Address" required />
+		<textarea name="message" placeholder="Write your message here..." maxlength="500" required > </textarea>
```
Comment (gtekle) [inline index.html:278]: - [ ] The required validation for the textarea doen't seem to work. Kindly check it.

**Review by gtekle — APPROVED** [review]
Hi @peterboro :wave: , 

# STATUS: APPROVED :tada: :tada: :tada: :100: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approved-stamp-02

## Highlights
:heavy_check_mark: Good use of Github flow
:heavy_check_mark: Meaningful commit messages
:heavy_check_mark: Descriptive pull request summary
:heavy_check_mark: Linters setup correctly
:heavy_check_mark: Nice UI

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**